src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,8 +1,166 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import core.basesyntax.service.UserHasNoLoginException;
+import core.basesyntax.service.UserNullException;
+import core.basesyntax.service.UserPasswordNullException;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
 public class HelloWorldTest {
+    private static RegistrationService registrationService;
+    private User user;
+
+    @BeforeAll
+    static void initial() {
+        registrationService = new RegistrationServiceImpl();
+        Storage.people.clear();
+    }
+
+    @BeforeEach
+     void createValidUser() {
+        user = new User();
+        user.setAge(42);
+        user.setLogin("labudabudapta");
+        user.setPassword("motorama");
+    }
+
+    @Test
+    void register_addSomeUsersWithUniqueLoginsToDatabase_Ok() {
```

Comment:
````suggestion
    void register_addSomeUsersWithUniqueLoginsToDatabase_ok() {
````
Each part must start with a lowercase letter